### DISCUSSION TITLE
Π2019223/ Παραδοτέο 6

### DISCUSSION THREAD

**Discussion by PetroulaStylianou** [courses-ionio/sw#1523] [Show and tell]
Σε αυτό το παραδοτέο μας ζητήθηκε να κάνουμε μία άσκηση γραμμής εντολών με ένα λογισμικό το οποίο θα είναι χωρίς systemd. Το λογισμικό που χρησιμοποίησα ήταν το Void Linux και για άσκηση γραμμής εντολών επέλεξα την άσκηση search the web from the terminal.

[Αναφορά](https://github.com/PetroulaStylianou/sw/blob/2019223/projects/2019223/README.md)

**Comment by p20sami** [comment]
Για το 6ο παραδοτεο η άσκηση που πρέπει να γινει είναι το custom static blog generator όπως λέει και στον πινακα

**Reply by PetroulaStylianou** [reply]
Ευχαριστώ για την υπενθύμιση, θα διορθωθεί άμεσα.
